shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -436,6 +438,23 @@ - (nullable id)accessibilityElementAtIndex:(NSInteger)index {
     return _semanticsObject;
   }
   SemanticsObject* child = [_semanticsObject children][index - 1];
+
+  // This 'if' block handles adding accessibility support for the embedded platform view.
+  // We first check if the child is a semantic node for a platform view.
+  // If so, we add the platform view as accessibilityElements of the child.
+  shell::FlutterPlatformViewsController* flutterPlatformViewsController =
+      _bridge.get()->flutter_platform_views_controller();
+  if (child.node.platformViewId > -1 && flutterPlatformViewsController) {
+    NSObject<FlutterPlatformView>* platformViewProtocolObject =
+        flutterPlatformViewsController->GetPlatformViewByID(child.node.platformViewId);
+    UIView* platformView = [platformViewProtocolObject view];
+    child.accessibilityElements = @[ platformView ];
+    return child;
+  } else if (child.accessibilityElements.count > 0 &&
+             [child.accessibilityElements.firstObject isKindOfClass:[UIView class]]) {
```

Comment:
Why do we need to do the clearing in two different places? It looks like if you do the clearing in the other place this will always be cleared here already?

Reply:
oh, forgot to move this testing code out. going to remove it.